tarok/test/tarok_utils.h

```diff
@@ -17,4 +18,7 @@ bool AllActionsInOtherActions(
     const std::vector<open_spiel::Action>& actions,
     const std::vector<open_spiel::Action>& other_actions);
 
+open_spiel::Action ActionFromCardName(std::string short_name,
```

Comment:
Did you mean `long_name` instead of `short_name`?

Reply:
Yep, my mistake. At first I used short_name but then changed it to long_name since it's more readable.